### ISSUE TITLE
Removal of Identity:0 in Output's name

### ISSUE THREAD

**Issue by abc3698** [onnx/keras-onnx#507]
Hi, I'm trying to convert to tensorRT int8 Model using onnx made by keras2Onnx.

My environment is as below:
python=3.7, keras2onnx=1.7, tensorflow=2.2.0, onnx=1.7, onnxconverter_common=1.7

My simple test onnx code is :
~~~
import os
os.environ['TF_KERAS'] = '1'
from tensorflow.keras.layers import Conv2D, MaxPooling2D, Dropout, Flatten, Dense, BatchNormalization, LeakyReLU, Input
from tensorflow.keras import Model
import keras2onnx

input = Input(batch_shape=(1, 28, 28, 1))

x = Conv2D(32, kernel_size=(5, 5), strides=(1, 1), padding='same')(input)
x = BatchNormalization(epsilon=0.001)(x)
x = LeakyReLU()(x)
x = MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)

x = Conv2D(64, (2, 2), padding='same')(x)
x = BatchNormalization(epsilon=0.001)(x)
x = LeakyReLU()(x)
x = MaxPooling2D(pool_size=(2, 2))(x)

x = Dropout(0.25)(x)
x = Flatten()(x)
x = Dense(1000, activation='relu')(x)
x = Dropout(0.5)(x)
x = Dense(10, activation='softmax')(x)

model = Model(input, x)

model.summary()

output_model_path = "mnist.onnx"
onnx_model = keras2onnx.convert_keras(model, model.name, target_opset=7)
keras2onnx.save_model(onnx_model, output_model_path)
~~~

I converted to tensorRT int8 Model and got an error like below:

~~~
[TensorRT] WARNING: Missing dynamic range for tensor leaky_re_lu/Identity:0, expect fall back to non-int8 implementation for any layer consuming or producing given tensor
[TensorRT] WARNING: Missing dynamic range for tensor leaky_re_lu/Identity:0_pooling0, expect fall back to non-int8 implementation for any layer consuming or producing given tensor
[TensorRT] WARNING: Missing dynamic range for tensor leaky_re_lu_1/Identity:0, expect fall back to non-int8 implementation for any layer consuming or producing given tensor
[TensorRT] WARNING: Missing dynamic range for tensor leaky_re_lu_1/Identity:0_pooling0, expect fall back to non-int8 implementation for any layer consuming or producing given tensor
[TensorRT] WARNING: Missing dynamic range for tensor flatten/Identity:0, expect fall back to non-int8 implementation for any layer consuming or producing given tensor
[TensorRT] WARNING: Missing dynamic range for tensor dense/Identity:0, expect fall back to non-int8 implementation for any layer consuming or producing given tensor
~~~

Above the error occurred only on nodes with Identity:0. But i've not found Identity:0 mean.
So what does "nodes with Identity:0" mean  and is any a way i remove Identity:0 in output names?

Thanks

**Comment by jiafatom** [comment]
Thanks for raising this. This issue is of two parts. keras->onnx, this is the scope of keras2onnx. Is the onnx model itself working on CPU? If yes, then onnx->tensorRT, this is another issue.
